.github/workflows/qa-azure.yaml

```diff
@@ -71,8 +77,22 @@ jobs:
           Import-PfxCertificate -Password $pwd -CertStoreLocation Cert:CurrentUser\My -FilePath certificate\certificate.pfx
       - name: Build Bundle
         run: |
+          # Set up the full version (displayed in logs)
+          $env:UP4W_FULL_VERSION=$(go run .\tools\build\compute_version.go)
+          
+          # Set up the numeric version (used in the AppxManifest)
+          $UP4W_VERSION=$(go run .\tools\build\compute_version.go --numeric)
```

Comment:
hum, so we have something different between the agent components themselves and the version set in the package.

Do you think we can write something in the some description section of the package, so that we have the exact version with the git sha?

Reply:
The difference is the tail end:
```
v0.1.0-7-gdeadbeef-dirty
0.1.0
```

Reply:
You probably miss the second part of my remark which was: can we write it in some part of package description which is readable once we install this manually?